### DISCUSSION TITLE
Different audio and video streams

### DISCUSSION THREAD

**Discussion by frosty-sh** [jhomlala/betterplayer#327] [Q&A] [answered]
Hi, I really appreciate the work you have done here. I have a question for the community since I don't have much experience with the plugin and I haven't been able to find it in the docs. Is there any way to list different audio streams for a video stream if they are not embedded into the video? I tried making a solution myself where I would to sync 2 players based and hide the one with the audio stream but it ended up being to clanky.

**Comment by jhomlala** [comment, marked as answer]
You can't play 2 different audios for one player. But you may try creating 2 players where one will be invisible. You can sync them with events: https://github.com/jhomlala/betterplayer#listen-to-video-events.
